src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/LanguageServices/Handlers/AdditionalFilesItemHandler.cs

```diff
@@ -48,7 +48,7 @@ private void AddToContextIfNotPresent(IWorkspaceProjectContext context, string f
             if (!_paths.Contains(fullPath))
             {
                 logger.WriteLine("Adding additional file '{0}'", fullPath);
-                context.AddAdditionalFile(fullPath, isActiveContext);
+                context.AddAdditionalFile(fullPath, folderNames: [], isActiveContext);
```

Comment:
What is this fixing? Can we explain a bit on why we adding folderNames here?

Reply:
The new Roslyn package deprecated some methods we were using. Following the message and reading through their code, I believe this is the correct alternative.